packages/components/src/components/input/component.tsx

```diff
@@ -46,7 +46,9 @@ export class KolInput implements Props {
 	}
 
 	public render(): JSX.Element {
-		const hasError = !this._readOnly && typeof this._error === 'string' && this._error.length > 0 && this._touched === true;
+		const isMessageValidError = Boolean(this._msg?._type === 'error' && this._msg._description && this._msg._description?.length > 0);
+		const hasError = !this._readOnly && isMessageValidError && this._touched === true;
+		const showFormFieldMsg = Boolean(hasError || (this._msg?._type !== 'error' && this._msg?._description));
 		const hasExpertSlot = showExpertSlot(this._label);
 		const hasHint = typeof this._hint === 'string' && this._hint.length > 0;
 		const useTooltopInsteadOfLabel = !hasExpertSlot && this._hideLabel;
```

Comment:
Zeile 61 ... dort den msg.type verwenden

Reply:
Verstehe ich leider nicht, was ist hier falsch?

In L61 wird indirekt der message.type verwendet, aber aber kombiniert mit den anderen Bedingungen, die es schon gab.